editor/Dockerfile

```diff
@@ -119,22 +119,39 @@ RUN [ `echo $version-$module | grep -v '^\(2018.3\|2018.4\).*-android'` ] && exi
     echo ''; \
     echo 'xvfb-run -ae /dev/stdout "$UNITY_PATH/Editor/Unity" -batchmode "$@"'; \
   } > /usr/bin/unity-editor \
-  && chmod +x /usr/bin/unity-editor
+  && chmod +x /usr/bin/unity-editor \
+  && { \
+    echo "export ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}"; \
+    echo "export ANDROID_HOME=${ANDROID_HOME}"; \
+    echo "export ANDROID_NDK_HOME=${ANDROID_NDK_HOME}"; \
+    echo "export JAVA_HOME=${JAVA_HOME}"; \
+    echo "export PATH=${PATH}"; \
+  } > ~/.bashrc
 
 #=======================================================================================
 # [2019.x/2020.x/2021.x-android] Support usage of installed Android SDKs outside of editor
 #=======================================================================================
-RUN [ `echo $version-$module | grep -v '^20(?!18).*-android'` ] && exit 0 || : \
+RUN [ `echo $version-$module | grep -vP '^20(?!18).*-android'` ] && exit 0 || : \
+  \
+  # Install jq
+  && apt-get update -qq \
+  && apt-get install -qq -y --no-install-recommends jq \
```

Comment:
Actually install `jq` which is used to parse Unity `modules.json` file.

Reply:
I have never seen `jq` used by Unity, do you have a source for this?

If this is used to parse Unity `modules.json`, should it not be installed for all versions, instead of making it part of the android specific fixes for 2019.x through 2021.x?

And if so, we should probably add it to the base image instead of the editor image, so we can also use it in Hub.

In case we're just adding it as a nice to have utility without Unity Editor actually needing it, we should check what the size of the installation with `--no-recommends` is. (e.g. 10MB+ just for parsing json would not be useful for CI images).

Reply:
It is currently only used by that particular android setup block. I'll just remove it after use.